### PR TITLE
Licence restrictions for Windows Update device

### PR DESCRIPTION
**Pull request by thomassa**

### CODE REVIEW

**Comment by thomassa** [comment]
Two alternative implementations to exempt `import` from the check; let's settle on one in code-review and I'll remove the other.

File: ocaml/xapi/xapi_vm.ml [hunk 1]
```diff
@@ -1066,6 +1061,11 @@ let query_services ~__context ~self =
 	raise (Api_errors.Server_error(Api_errors.not_implemented, [ "query_services" ]))
 
 let assert_can_set_auto_update_drivers ~__context ~self ~value =
+	if value && not (Db.VM.get_auto_update_drivers ~__context ~self)
```
Comment (robhoes) [inline ocaml/xapi/xapi_vm.ml]: We can probably just do `if value then` and save ourselves a DB get call.

**Comment by simonjbeaumont** [comment]
Removing the 'reviewed' flag as the 'needs updating' flag is present. I realise that these are not linguistically mutually exclusive but I think the green reviewed flag means 'good to go' and the orange 'needs updating' flag means just that. They both imply a review has taken place. If it helps we can consider renaming reviewed to approved or similar.

**Comment by thomassa** [comment]
Discussed this yesterday with @robhoes; now I've made the changes and it's ready for review again.

**Comment by robhoes** [comment]
Looks good to me. @thomassa said he will make one more change, and then I'd say :shipit: .

**Comment by thomassa** [comment]
Change made. I'll just wait for the auto-tests to pass again, then merge.

**Pull request merged**